static/app/components/tooltip.tsx

```diff
@@ -72,6 +72,12 @@ type Props = DefaultProps & {
    * Should be set to true if tooltip contains unisolated data (eg. dates)
    */
   disableForVisualTest?: boolean;
+
+  /**
+   * Force the tooltip to be visible without hovering
+   */
+  show?: boolean;
```

Comment:
Maybe change this to `forceShow` for clarity? `show` by itself doesn't really indicate that the tooltip state will be semi-controlled externally?

Reply:
Yeah, `forceShow` sounds pretty good tome